src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,19 @@
-export const CommentList = () => (
-  <>Put the list here</>
-);
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  if (comments.length === 0) {
+    return (
+      <>
+        <hr />
+        <b data-cy="NoCommentsMessage">No comments yet</b>
+      </>
+    );
+  }
+
+  return comments.map(comment => (
+    <div className="CommentList">
+      <CommentInfo comment={comment} key={comment.id} />
+    </div>
```

Comment:
Need to add key to the parent element
```suggestion
  return comments.map(comment => (
    <div className="CommentList" key={comment.id}>
      <CommentInfo comment={comment} />
    </div>
```